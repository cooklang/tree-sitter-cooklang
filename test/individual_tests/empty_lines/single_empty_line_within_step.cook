Mix @flour{200%g} and @water{100%ml}.

Continue mixing until smooth.