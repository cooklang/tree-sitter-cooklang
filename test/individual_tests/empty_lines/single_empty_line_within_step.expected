(recipe 
  (step 
    (ingredient (name) (amount (quantity) (units)))
    (ingredient (name) (amount (quantity) (units)))))